docs/index.mdx

```diff
@@ -8,24 +8,80 @@ slug: /
 
 # Meet Fused
 
-Fused Workbench is an end-to-end cloud platform for data analytics. Fused revolves around User Defined Functions ([UDFs](/core-concepts/why/)), which are Python functions that can be called via HTTP requests.
+Fused Workbench is an end-to-end cloud platform for data analytics. 
+
+The core principle behind Fused are Python functions that can be run via HTTP requests, called **User Defined Functions ([UDFs](/core-concepts/why/))**.
+
+<!--
 
 <img src="https://fused-magic.s3.us-west-2.amazonaws.com/main_marketing_website/product_diagram_october2024.png" alt="product_diagram_october2024" style={{}} />
 
-## Why Fused?
+-->
+
+## First look
+
+_We're going to create a simple User Defined Function and return it's results in a Google Sheets. Feel free to follow along by [first making sure you can login](/user-guide/login)._
+
+**User Defined Functions** are Python functions that run on the Fused backend with a HTTP endpoint. 
+
+Here's an example, we're going to load New York City metro station, taken from a GeoJSON directly hosted on Github [from the `folium-example-data` public repo](https://github.com/python-visualization/folium-example-data):
```

Comment:
```suggestion
As an example, we're going to load New York City metro station, taken from a GeoJSON directly hosted on Github [from the `folium-example-data` public repo](https://github.com/python-visualization/folium-example-data):
```

Reply:
Suggest avoiding use of the full "User Defined Function" after the acronym has been introduced.

Reply:
"a Google Sheets" -> "a Google Sheet"